lib/Dialect/LinalgExt/IR/TiledOpInterface.cpp

```diff
@@ -305,9 +306,11 @@ void IREE::LinalgExt::registerTiledOpInterfaceExternalModels(
     DialectRegistry &registry) {
   LLVM_DEBUG(
       { llvm::dbgs() << "Adding external models of tiled op interface\n"; });
-  registry
-      .addOpInterface<tensor::ExtractSliceOp, ExtractSliceTiledOpInterface>();
-  registry.addOpInterface<tensor::InsertSliceOp, InsertSliceTiledOpInterface>();
+  registry.addExtension(+[](MLIRContext *ctx,
```

Comment:
I see `addExtension(+[](`, is the `+` a typo?

Reply:
No it is not: https://stackoverflow.com/questions/18889028/a-positive-lambda-what-sorcery-is-this